data/json/items/tool/workshop.json

```diff
@@ -694,7 +694,8 @@
     "color": "brown",
     "qualities": [ [ "HAMMER", 2 ] ],
     "flags": [ "NONCONDUCTIVE", "BELT_CLIP" ],
-    "melee_damage": { "bash": 9 }
+    "melee_damage": { "bash": 9 },
+    "weapon_category": [ "BATONS" ]
```

Comment:
```suggestion
```
One third of the length of batons.

Reply:
I would like to refer to andrewhr's comment. If the makeshift_hammer and the primitive_hammer are too short for BATONS, then hammer requires a revision too.

Reply:
Hammer (I'm assuming a typical carpentry or claw hammer) shouldn't really be considered a baton either no.